frontend/src/environments/environment.dev.ts

```diff
@@ -0,0 +1,15 @@
+export const environment = {
+  production: false,
+  environment_cloud: false,
+  environment_desktop: false,
+  environment_name: "dev",
+  popup_source_auth: false,
+
+  lighthouse_api_endpoint_base: 'https://lighthouse.fastenhealth.com/sandbox',
+
+  //used to specify the couchdb server that we're going to use (can be relative or absolute). Must not have trailing slash
+  couchdb_endpoint_base: '/database',
```

Comment:
this key is actually unnecessary in all the files. its dead code from an earlier version of fasten.